java/pom.xml

```diff
@@ -60,6 +60,7 @@
   </modules>
 
   <properties>
+    <java.version>1.8</java.version>
```

Comment:
Oh, could you put this new property in a sorted order? CI seems to complain at the order.

Reply:
Let me fix this~